api-reference/10 UI Widgets/dxScheduler/1 Configuration/recurrenceEditMode.md

```diff
@@ -9,16 +9,30 @@ default: 'dialog'
 Specifies the edit mode for recurring appointments.
 
 ---
-This option accepts the following values.
+This option accepts the following values:
 
-- "dialog"  
- Displays a dialog that suggests to a user to choose between editing the entire series or only the current appointment.
+- *"series"*  
+ Enables a user to edit only the entire appointment series.
 
-- "series"  
- Enables an end-user to edit only the entire appointment series.
+- *"occurrence"*  
+ Enables a user to edit only the individual appointment instance.
 
-- "occurrence"  
- Enables an end-user to edit only the current appointment.
+- *"dialog"*
+ Displays a dialog that suggests to a user to choose between editing the entire series or only the individual instance.
+
+Changes made to recurring appointment instance and series are handled differently.
+
+If a user edits a recurring appointment instance, two actions are performed on the data objects:
+
+- The series' data object is updated.         
+The **Scheduler** updates the field specified by [recurrenceExceptionExpr](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/recurrenceExceptionExpr.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#recurrenceExceptionExpr'), thus adding the edited instance to exceptions. The [onAppointmentUpdating](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/onAppointmentUpdating.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentUpdating') and [onAppointmentUpdated](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/onAppointmentUpdated.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentUpdated') event handlers are executed.
+
+- A new data object is created.       
+This object contains the edited instance's data. The [onAppointmentAdding](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/onAppointmentAdding.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentAdding') and [onAppointmentAdded](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/onAppointmentAdded.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentAdded') event handlers are executed.
+
+When an individual appointment instance is deleted, the **Scheduler** adds it to the entire appointment series exceptions by updating the field that **recurrenceExceptionExpr** specifies. Because this is an update, the **onAppointmentUpdating** and **onAppointmentUpdated** event handlers are executed instead of [onAppointmentDeleting](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleting) and [onAppointmentDeleted](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleted).
```

Comment:
```suggestion
When a user deletes an instance, the **Scheduler** adds it to series' exceptions by updating the field specified by **recurrenceExceptionExpr**. Because this is an update, the **onAppointmentUpdating** and **onAppointmentUpdated** event handlers are executed instead of [onAppointmentDeleting](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleting) and [onAppointmentDeleted](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleted).
```

Reply:
"...by updating the field specified by **recurrenceExceptionExpr**..." -> "...by updating the field specified **in** **recurrenceExceptionExpr**..." ?

Reply:
I don't see much difference but okay, we can use "in" to remove the repetition of "by".